test/unit/core.js

```diff
@@ -464,8 +464,11 @@ QUnit.test( "isNumeric", function( assert ) {
 	assert.ok( t( -16 ), "Negative integer number" );
 	assert.ok( t( 0 ), "Zero integer number" );
 	assert.ok( t( 32 ), "Positive integer number" );
+	assert.ok( t( "0b111110" ), "Binary integer literal" );
```

Comment:
The eslint problem can be fixed with inline comments like `// j`, but the [JSHint docs](http://jshint.com/docs/options/) say that the `exnext` option is deprecated and `esversion: 6` should be used instead. So 
```js
// jshint esversion: 6
   ... code that uses es6 ...
// jshint esversion: 5
```
I don't really  like explicitly setting esversion back to 5 since it makes an assumption, is there a way to simply revert back to the previous setting? Is the scope of this change only the current test/block in JS?